README.md

```diff
@@ -1,80 +1,90 @@
 # multi-cluster-traffic-controller
-// TODO(user): Add simple overview of use/purpose
 
-## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
-
-## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
-
-### Running on the cluster
-1. Install Instances of Custom Resources:
-
-```sh
-kubectl apply -f config/samples/
-```
-
-2. Build and push your image to the location specified by `IMG`:
-	
-```sh
-make docker-build docker-push IMG=<some-registry>/multi-cluster-traffic-controller:tag
-```
+## Description:
+When deploying the multi cluster traffic controller using the make targets the following will be created: 
+* Kind cluster(s)
+* Gateway API CRDs in the control plane cluster
+* Ingress controller
+* Cert manager
+* External DNS
```

Comment:
We should remove this again, got added back for some reason, external dns is not being used.

Reply:
Cool